emperor/support_files/js/view.js

```diff
@@ -694,6 +856,19 @@ DecompositionView.prototype.setColor = function(color, group) {
     });
     cloud.geometry.attributes.color.needsUpdate = true;
   }
+  else if (UIState['view.viewType'] == 'parallel-plot' &&
+           this.decomp.isScatterType()) {
+    var lines = this.markers[0];
+    color = new THREE.Color(color);
+    var numPoints = (this.decomp.dimensions * 2 - 2);
+    group.forEach(function(plottable) {
+    var startIndex = plottable.idx * numPoints;
+    var endIndex = (plottable.idx + 1) * numPoints;
+    for (var i = startIndex; i < endIndex; i++)
+        lines.geometry.attributes.color.setXYZ(i, color.r, color.g, color.b);
```

Comment:
This line looks over indented, can you remove two of the leading spaces?

Reply:
Done